ember-cli-build.js

```diff
@@ -5,6 +5,9 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function(defaults) {
   let app = new EmberAddon(defaults, {
     // Add options here
+    'ember-test-selectors': {
```

Comment:
Yes that has been changed since yesterday 
https://github.com/lblod/ember-submission-form-fields/pull/5/commits/40936c422fec7b58aa946daf247b5de2b87ba84f